image_service/urls.py

```diff
@@ -0,0 +1,34 @@
+"""
+URL configuration for image_service project.
+
+The `urlpatterns` list routes URLs to views. For more information please see:
+    https://docs.djangoproject.com/en/4.2/topics/http/urls/
+Examples:
+Function views
+    1. Add an import:  from my_app import views
+    2. Add a URL to urlpatterns:  path('', views.home, name='home')
+Class-based views
+    1. Add an import:  from other_app.views import Home
+    2. Add a URL to urlpatterns:  path('', Home.as_view(), name='home')
+Including another URLconf
+    1. Import the include() function: from django.urls import include, path
+    2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
+"""
+from django.conf.urls.static import static
+from django.contrib import admin
+from django.urls import path, include
+from rest_framework import routers
+
+from image_manager.api.views import ImageViewSet
+from image_service import settings
+
+router = routers.DefaultRouter()
+router.register(r'images', ImageViewSet, basename='image')
+
+urlpatterns = [
+    path('admin/', admin.site.urls),
+    path('api/', include(router.urls)),
+]
+
+if settings.DEBUG:
```

Comment:
In a production environment, we should serve media files through a web server like Nginx. Nginx is optimized for this task and can handle large numbers of simultaneous connections with a low memory footprint.